modules/programs/nushell.nix

```
@@ -6,6 +6,26 @@ let
 
   cfg = config.programs.nushell;
 
+  configDir = if pkgs.stdenv.isDarwin then
+    "Library/Application Support/nushell"
+  else
+    "nushell";
```

Can use `home.file` for both Darwin and GNU/Linux by giving the full path here:

```suggestion
    "${config.xdg.configHome}/nushell";
```